src/lib/classes/lsp8-identifiable-digtal-asset.spec.ts

```diff
@@ -306,8 +307,19 @@ describe('LSP8IdentifiableDigitalAsset', () => {
         '0x591c236982b089Ad4B60758C075fA50Ec53CD674',
       ];
 
+      let lspFactory: LSPFactory;
+
+      it('set UP address in creator array', async () => {
+        lspFactory = new LSPFactory(provider, signer);
+        const contracts = await lspFactory.UniversalProfile.deploy({
+          controllerAddresses: [controllerAddress],
+        });
+
+        const universalProfileAddress = contracts.LSP0ERC725Account.address;
+        creators.push(universalProfileAddress);
+      });
+
```

Comment:
looks like duplicated code with lsp7 test - where is the assert/expect?

If you use this code a lot maybe you can move it to a utils file.